onnxruntime/test/framework/tunable_op_test.cc

```diff
@@ -15,12 +15,6 @@
 #pragma comment(lib, "WinMM.lib")
```

Comment:
Why this line of code exists?

Reply:
1. The test still runs on Windows when RTTI is available
2. I use std::this_thread::sleep_for to mock slow and fast kernel impl
3. On Windows, the the wake up timing is not guaranteed if you don't call some system function.

This just makes the symbol for system call available on linking for that purpose.

Reply:
You may use other timers. You don't have to use https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleepex. We don't want to have an extra dependency for the CPU build.

Reply:
In the [Remarks section](https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-sleepex#remarks) it says
> Note that a ready thread is not guaranteed to run immediately.
This is the same as `std::this_thread::sleep_for`.

> The system clock "ticks" at a constant rate. To increase the accuracy of the sleep interval, call the timeGetDevCaps function to determine the supported minimum timer resolution and the timeBeginPeriod function to set the timer resolution to its minimum. Use caution when calling timeBeginPeriod, as frequent calls can significantly affect the system clock, system power usage, and the scheduler. If you call timeBeginPeriod, call it one time early in the application and be sure to call the timeEndPeriod function at the very end of the application.
It still requires to call `timeBeginPeriod` to avoid flaky test on Windows. So it actually falls back to current situation, `timeBeginPeriod` is the system call I use in this test. 

Considering using the native timer API won't improve the situation here, so I will not change as suggested.

Reply:
I said: you don't have to use the function. There might be other APIs you can use.

Reply:
I do not have desire to search for a Windows API to solely avoid an additonal linker input for some testing binary. If you do not have viable alternative as drop-in replacement, then I will leave this open and will not update on it.

Reply:
I'm not able to answer every question you have,  but we have many Windows developers in our team.  They develop Windows. They know it very well. We could ask them.

Reply:
So I removed the sleep mock all together. The mocked kernel has data dependency, so repeatly launch will be sufficient to make it slower and the compiler is not smart enought to fuse (or partially evaluate) that portion of code to optimize it away. I think I can mock slow and fast kernel in the new way reliably without using system call anymore.